.prettierignore

```diff
@@ -5,6 +5,8 @@
 .gitattributes
```

Comment:
Not related to this line/file:
Changing back to Lisk application gives error

![image](https://user-images.githubusercontent.com/8784876/237056806-abc8aae6-11c8-46a3-9385-54f1df15b355.png)

Reply:
But first time i open the modal it works. But maybe this is related to the network being flunky?

Reply:
yea.. if the network is flaky it will show the error, and if it fetches the metadata then it shouldn't show error.